pkg/dependency/parser/java/jar/sonatype/log.go

```diff
@@ -9,24 +9,24 @@ type logger struct{}
 func (logger) Error(msg string, keysAndValues ...interface{}) {
 	// Use Debugw to suppress errors on failure
 	if msg == "request failed" {
-		log.Logger.Debugw(msg, keysAndValues...)
+		log.Debug(msg, keysAndValues...)
 		return
 	}
-	log.Logger.Errorw(msg, keysAndValues)
+	log.Error(msg, keysAndValues)
 }
 
 func (logger) Info(msg string, keysAndValues ...interface{}) {
-	log.Logger.Infow(msg, keysAndValues...)
+	log.Info(msg, keysAndValues...)
 }
 
 func (logger) Debug(msg string, keysAndValues ...interface{}) {
 	// This message is displayed too much
 	if msg == "performing request" {
 		return
 	}
-	log.Logger.Debugw(msg, keysAndValues...)
+	log.Debug(msg, keysAndValues...)
```

Comment:
It seems to me that the logger from the `slog` package implements the interface from the `go-retryablehttp` package. You can also use a logging prefix in this package.

Reply:
Nice catch! Updated in https://github.com/aquasecurity/trivy/pull/6466/commits/192c8bc51ca40c35920b5360c9c8d2144b642605